sktime/forecasting/neuralforecast.py

```diff
@@ -163,7 +161,7 @@ class NeuralForecastRNN(_NeuralForecastAdapter):
         # inherited from _NeuralForecastAdapter
         # estimator type
         # --------------
-        "python_dependencies": ["neuralforecast>=1.6.4"],
+        "python_dependencies": ["neuralforecast>=1.7.0"],
```

Comment:
This is a potentially breaking change. Users who are using neuralforecast 1.6.4 with sktime currently will not longer be able to use this estimator any more. Especially if they don't want to customose optimiser, it will be unnecessary. Please revert this change.